topics/functions-intro.md

```diff
@@ -372,9 +361,9 @@ And your Valkey log file should have lines in it that are similar to:
 ## Functions in cluster
 
 As noted above, Valkey automatically handles propagation of loaded functions to replicas.
-In a Valkey Cluster, it is also necessary to load functions to all cluster nodes. This is not handled automatically by Valkey Cluster, and needs to be handled by the cluster administrator (like module loading, configuration setting, etc.).
+In a [Valkey Cluster](cluster-tutorial.md), it is also necessary to load functions to all primaries in the cluster. This is not handled automatically by Valkey Cluster, and needs to be handled by the cluster administrator (like module loading, configuration setting, etc.).
```

Comment:
can we just say "In a cluster"... I don't think people will think they're going to load it into a mysql cluster :)

Reply:
```suggestion
In a [cluster](cluster-tutorial.md), it is necessary to load functions to all primaries. ```

Reply:
LGTM